### PR TITLE
Search multiple paths for authenticator_plugins

### PR DESCRIPTION
**Pull request by john-westcott-iv**
Allows setting multiple paths from ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIX

### CODE REVIEW

File: ansible_base/authenticator_plugins/utils.py [hunk 1]
```diff
@@ -5,33 +5,32 @@
 from django.conf import settings
 
 logger = logging.getLogger('ansible_base.authenticator_plugins.utils')
-setting = 'ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIX'
+setting = 'ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIXS'
```
Comment (relrod) [inline ansible_base/authenticator_plugins/utils.py]:
```suggestion
setting = 'ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIXES'
```

File: ansible_base/tests/settings_overrides.py [hunk 2]
```diff
@@ -63,4 +63,4 @@
     "ansible_base.authentication.backend.AnsibleBaseAuth",
 ]
 
-ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIX = 'ansible_base.authenticator_plugins'
+ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIXS = ['ansible_base.authenticator_plugins']
```
Comment (relrod) [inline ansible_base/tests/settings_overrides.py]:
```suggestion
ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIXES = ['ansible_base.authenticator_plugins']
```

File: docs/authentication.md [hunk 3]
```diff
@@ -29,13 +29,13 @@ MIDDLEWARE = [
 Note: this must come before django.contrib.auth.middleware.AuthenticationMiddlware in order to have precedence over it. Otherwise a local user will be authenticated even if the user was destined for LDAP/Tacacs+/Radius/etc. 
 
 
-### ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIX
+### ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIXS
```
Comment (relrod) [inline docs/authentication.md]:
```suggestion
### ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIXES
```

File: docs/authentication.md [hunk 4]
```diff
@@ -29,13 +29,13 @@ MIDDLEWARE = [
...
 Next we need to setup the class prefix for the installed authenticator classes, this can be:
 ```
-ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIX = "ansible_base.authenticator_plugins"
+ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIXS = ["ansible_base.authenticator_plugins"]
```
Comment (relrod) [inline docs/authentication.md]:
```suggestion
ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIXES = ["ansible_base.authenticator_plugins"]
```

File: docs/authentication.md [hunk 5]
```diff
@@ -110,6 +110,6 @@ django-ansible-base comes with many types of authenticators which can be found i
 
 If you wanted to remove authenticators from your application there are two ways to do this:
 1. Remove the unwanted files from your ansible_base installation.
-2. Create a new class directory in your application and only add in the authenticators you care about and then set `ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIX` to be the prefix for your class.
+2. Create a new class directory in your application and only add in the authenticators you care about and then set `ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIXS` to be the prefix for your class.
```
Comment (relrod) [inline docs/authentication.md]:
```suggestion
2. Create a new class directory in your application and only add in the authenticators you care about and then set `ANSIBLE_BASE_AUTHENTICATOR_CLASS_PREFIXES` to be the prefix for your class.
```

Comment (john-westcott-iv) [reply docs/authentication.md]: Thanks for reminding me to add my spell checker pre-commit user hook :)

**Review by relrod — APPROVED** [review]

**Pull request merged**